decision_service/include/ds_api_status.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <string>
+
+
+namespace decision_service {
+
+	namespace error_code {
+		//success code
+		const int success = 0;
+
+		//error code
+		const int invalid_argument = 1;
+		const int background_queue_overflow = 2;
+		const int eventhub_http_generic = 3;
+		const int eventhub_http_bad_status_code = 4;
+	}
+
+
+	class api_status {
+	public:
+		int get_error_code() const;
+		void set_error_code(int);
```

Comment:
>void set_error_code(int); [](start = 2, length = 25)

should be private

Reply:
removed method

---
In reply to: [186733796](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186733796) [](ancestors = 186733796)